comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -172,7 +173,9 @@
       <!--
         Do not edit this section manually and use ./comps_doc.sh script
       -->
+      <packagereq type="default">rubygem-smart_proxy_acd</packagereq>
       <!--RGD-START-->
+      <packagereq type="default">rubygem-smart_proxy_acd-doc</packagereq>
```

Comment:
the smart proxy is SCL'ed these days too, so it should be `tfm-rubygem-…` and then also sorted accordingly ;)